manage.py

```diff
@@ -101,6 +102,8 @@ def force_build_static_site():
         print("Build is disabled at the moment. Set BUILD_SITE to true to enable")
 
 
+# Run this command with the parameter default_user_password to set up additional default user accounts
+# e.g. ./manage.py pull_prod_data --default_user_password=P@55w0rd
 @manager.command
 def pull_prod_data(default_user_password=None):
```

Comment:
Any reason for not adding a default value for this parameter? 'Security', presumably ... but probably it's fine if the default is only used locally?

Reply:
onditional on password being suppliedCreating default users is conditional on password being supplied. I thought we might want to be able to run it without creating any extra users. Though now I think about it, I'm not sure what circumstance that would be.

So yes, you're right... it's for "security" reasons.  Don't question it unless you're an enemy of "security".

Reply:
🚫 Security! 🚫